modules/viewing-pipeline-versions.adoc

```diff
@@ -0,0 +1,33 @@
+:_module-type: PROCEDURE
+
+[id="viewing-pipeline-versions_{context}"]
+= Viewing pipeline versions
+
+[role='_abstract']
+You can view the details of pipeline versions that you have uploaded to {productname-long}, such as its creation date and its runs.
+
+.Prerequisites
+* You have installed the OpenShift Pipelines operator.
+* You have logged in to {productname-long}.
+ifndef::upstream[]
+* If you are using specialized {productname-short} groups, you are part of the user group or admin group (for example, {oai-user-group} or {oai-admin-group}) in OpenShift.
+endif::[]
+ifdef::upstream[]
+* If you are using specialized {productname-short} groups, you are part of the user group or admin group (for example, {odh-user-group} or {odh-admin-group}) in OpenShift.
+endif::[]
+* You have previously created a data science project that is available and contains a pipeline server.
+* You have a pipeline available on an active and available pipeline server.
+
+.Procedure
+. From the {productname-short} dashboard, click *Data Science Pipelines* -> *Pipelines*.
++
+The *Pipelines* page opens.
+. From the *Project* list, select the relevant project that contains the pipeline versions you want to view.
+. Study the pipelines on the list.
+. Click *Expand* (image:images/rhoai-expand-icon.png[]) on the row containing the relevant pipeline.
```

Comment:
Suggest updating to 'the row containing the pipeline that you want to view versions for.'

Reply:
Fixed.